src/main/java/org/example/Interpolation.java

```diff
@@ -0,0 +1,46 @@
+package org.example;
+
+public class Interpolation {
```

Comment:
Dodajte dokumentacijo razreda in njegovi metod. Dokumentacija naj vsebuje kratek opis in če je smiselno opis argumentov in vrednosti, ki jih metoda vrne in primer klica metode.